src/banner-card.html

```diff
@@ -0,0 +1,61 @@
+<link rel="import" href="../bower_components/polymer/polymer.html">
+<link rel="import" href="../bower_components/app-layout/app-header/app-header.html">
+<link rel="import" href="../bower_components/app-layout/app-toolbar/app-toolbar.html">
+<link rel="import" href="../bower_components/paper-button/paper-button.html">
+<link rel="import" href="../bower_components/paper-card/paper-card.html">
+<link rel="import" href="../bower_components/paper-styles/typography.html">
+<link rel="import" href="shared-styles.html">
+
+<dom-module id="banner-card">
+    <template>
+        <style>
+            paper-card.banner {
+                @apply(--layout-horizontal);
+                margin: 15px 20px;
+            }
+
+            .banner-image {
+                width: 60%;
+                height: 300px;
+                background: url('http://www.quansershare.com/Images/submit-ad.png');
```

Comment:
load it from the page, it's better because the web cache make it lighter, I would even consider use [`<iron-image>`](https://www.webcomponents.org/element/PolymerElements/iron-image)